### PR TITLE
Update symfony.rst

### PR DESCRIPTION
**Pull request by andreybolonin**
https://symfony.com/releases

### CODE REVIEW

File: source/howto/symfony.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 .. |app| replace:: Symfony
-.. |mod| replace:: PHP 7.2.5+
+.. |mod| replace:: PHP 8.1+
```
Comment (javorszky) [inline source/howto/symfony.rst:2]: Hi there! I see that 8.1 is needed for Symfony's long term support release 6.4.9 that will be supported until Novemebr 2026.

The current stable release needs 8.2+.

I would suggest updating this to. Folks needing long term support are most probably already aware that php 8.1 is their requirement.

Comment (javorszky) [reply source/howto/symfony.rst:2]:
```suggestion
.. |mod| replace:: PHP 8.2+
```

**Review by javorszky — CHANGES_REQUESTED** [review]
Once you rebase over current main or otherwise resolve the conflicts and respond to the comment about version, I'm happy to merge this in.

I'm also open to leaving it at 8.1 if there's a strong argument for it.

Thank you for taking the time on this! 🙂

**Comment by andreybolonin** [comment]
@javorszky updated

**Review by javorszky — APPROVED** [review]

**Comment by javorszky** [comment]
> @javorszky updated

Thank you! Approved and merged :)

**Pull request merged**